configuration.md

```diff
@@ -0,0 +1,16 @@
+# Configuration
+
+Environment variables can be used to configure the project:
+
+## Database configuration
+
+* **PGHOST** (default `postgres`): The address at which the Postgres server can be reached
+* **PGPORT** (default `5432`): The port at which the Postgres server can be reached
+* **PGDATABASE** (default `lionweb_test`): The name of the Postgres database to be used within the Postgres server
+* **PGUSER** (default `postgres`): The username used to connect to the Postgres server
+* **PGPASSWORD** (default `lionweb`): The password used to connect to the Postgres server
+
+## Node application configuration
+
+* **NODE_PORT** (default `3005`): Port at which the lionweb repository can be reached
```

Comment:
https://github.com/LionWeb-io/lionweb-repository/blob/24ce3a4cfc5b0881bb8c39b6258763571ba5ac13/packages/core/src/server.ts#L39 suggests that 3005 is still hard-coded.